api/addons/v1beta1/v1beta2_condition_consts.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2024 The Kubernetes Authors.
+Copyright 2021 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2024 The Kubernetes Authors.
```
This should stay 2024